NEWS

```diff
@@ -117,13 +117,9 @@ PHP                                                                        NEWS
     (nielsdos)
 
 - GMP:
-  . Fixed floating point exception bug with gmp_pow when using
-    large exposant values. (David Carlier).
   . Fixed bug GH-16411 (gmp_export() can cause overflow). (cmb)
   . Fixed bug GH-16501 (gmp_random_bits() can cause overflow).
     (David Carlier)
-  . Fixed gmp_pow() overflow bug with large base/exponents.
-    (David Carlier)
```

Comment:
I don't think we should edit the NEWS file for prior entries, better to leave them and add new NEWS entries.